src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -832,7 +832,19 @@ namespace OpenLoco::ObjectManager
     // 0x00472031
     void unloadAll()
     {
-        call(0x00472031);
+        for (auto type = ObjectType::interfaceSkin; enumValue(type) <= enumValue(ObjectType::scenarioText); type = static_cast<ObjectType>(enumValue(type) + 1))
```

Comment:
I get what you're doing... But wouldn't it be more readable to just iterate an integer to `kMaxObjectTypes` instead? Then cast that into an ObjectType when calling `getMaxObjects`.

Reply:
okay i've reworked things